README.md

```diff
@@ -1,70 +1,97 @@
-# Getting Started with Create React App
+# Marvel Heros
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+> This app uses React and Redux, Along with React Router for data manipulation from a couple of web services. It is a SPA with the store object attached to the index.js component used by all of the components to make use and manupulate the data. The app contains data from the Marvel Developer API. Follows Javascript and CSS best practices. It renders information to the user on the main home page and passes the detail page with appropriate ID variable for the respective hero.
 
-## Available Scripts
 
-In the project directory, you can run:
+# Live Demo
 
-### `npm start`
+- [Netlify](https://musing-fermi-c78199.netlify.app/) 
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+# Video Description
 
```

Comment:
- [OPTIONAL] I would have been nicer, more professional if your face was showing on the video presentation. Please kindly give your presentation a face.